src/lib/public/board/data/teams.json

```diff
@@ -1,4 +1,16 @@
 [
+  {
+    "title": "General",
+    "id": "general",
+    "picture": "/assets/badges/general.svg",
+    "color": "var(--acm-blue)"
+  },
+  {
+    "title": "AI",
+    "id": "ai",
+    "picture": "/assets/badges/ai-team.svg",
+    "color": "var(--acm-emerald)"
```

Comment:
@EthanThatOneKid Why do we use `"color": "var(--acm-emerald)"` instead of  `"color": "var(--acm-ai-rgb)"`?

Reply:
> @/EthanThatOneKid Why do we use `"color": "var(--acm-emerald)"` instead of `"color": "var(--acm-ai-rgb)"`?

Our [`global.css`](https://github.com/EthanThatOneKid/acmcsuf.com/blob/8bd8c6aa1aad39b43e1d90c13482a6b2605419e6/static/global.css) file includes several styles used globally throughout <https://acmcsuf.com/>. One section of this file is the `:root { /* ... */ }` block which instantiates the values of our global CSS variables.

In <https://acmcsuf.com/>, the variables with the prefix `--acm-` are all expected to be valid CSS colors. The variables with the suffix `-rgb` are all expected to be tuples of 3 integers (0-255) the first representing the amount of red, the second representing green, and the third representing blue. Note: This is not 100% consistent, so it is recommended to reference `global.css` each time we are working with a global CSS variable.

The variables with the prefix `--acm-` are intended to be used in CSS code that expects a color.

The variables with the suffix `-rgb` are intended to be used to set a custom opacity. For example:
<https://github.com/EthanThatOneKid/acmcsuf.com/blob/8bd8c6aa1aad39b43e1d90c13482a6b2605419e6/src/lib/components/toaster/toaster.svelte#L55>

This was implemented quite some time ago, so I am open to any ways to improvement our occurrences of `-rgb` CSS variables.

CC @karnikaavelumani